Suggest to install without root permissions

apiraino opened the pull request:
I'd like to easily install `pass-otp` without requiring root access and respecting XDG base dir specs.

This is my attempt to do that without touching the code, rather playing a bit with env vars

apiraino committed:
update README

apiraino committed:
be more helpful

apiraino committed:
even more helpful

apiraino committed:
Use standard XDG bash completion dir

apiraino committed:
Fix extension dir env var

apiraino commented:
@tadfisher hi, pinging to check interest on this small PR :)

tadfisher commented:
@apiraino Can you check this against the Makefile changes that went in with #111 ?

apiraino committed:
remove now redundant  env var

apiraino committed:
Merge remote-tracking branch 'upstream/develop' into install-xdg-path

apiraino commented:
Hi @tadfisher I've cursory checked the diff. I think that PR makes redundant setting `$MANDIR` so I have removed it from this PR, thanks for the heads-up :+1: 

(I've also updated against your `develop` branch)
